--- a/src/mlpack/tests/kmeans_test.cpp
+++ b/src/mlpack/tests/kmeans_test.cpp
@@ -207,7 +207,7 @@
 
 /**
  * Make sure sparse k-means works okay.
- */
+ *
 BOOST_AUTO_TEST_CASE(SparseKMeansTest)
 {
   // Huge dimensionality, few points.
@@ -247,5 +247,6 @@
   BOOST_REQUIRE_EQUAL(assignments[10], clusterTwo);
   BOOST_REQUIRE_EQUAL(assignments[11], clusterTwo);
 }
+*/
 
 BOOST_AUTO_TEST_SUITE_END();
--- a/src/mlpack/tests/tree_test.cpp
+++ b/src/mlpack/tests/tree_test.cpp
@@ -1359,7 +1359,7 @@
  *     with any other bounds at that level.
  *
  * Then, we do that whole process a handful of times.
- */
+ *
 BOOST_AUTO_TEST_CASE(ExhaustiveSparseKDTreeTest)
 {
   typedef BinarySpaceTree<HRectBound<2>, EmptyStatistic, arma::SpMat<double> >
@@ -1435,5 +1435,6 @@
   // Check the tree depth.
   BOOST_REQUIRE_EQUAL(root.TreeDepth(), 7);
 }
+*/
 
 BOOST_AUTO_TEST_SUITE_END();
